frontend/models/feedback-task-model/global-question/globalQuestion.queries.js

```diff
@@ -0,0 +1,26 @@
+import { GlobalQuestion as GlobalQuestionModel } from "../global-question/GlobalQuestion.model";
+
+// UPSERT
+const upsertGlobalQuestion = (globalQuestion) => {
+  GlobalQuestionModel.insertOrUpdate({ data: globalQuestion });
+};
```

Comment:
Why we have upsertGlobalQuestion and upsertGlobalQuestions?

Reply:
upsertGlobalQuestion => upsert one single question in the globalQuestion entity
upsertGlobalQuestions => upsert a table of questions in the globalQuestion entity

Reply:
oups. done.